### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by aogriffiths**
Hi, 

I've made lots of improvements to the documentation. It would be great if you would like to merge these in.

Thanks
Adam

### CODE REVIEW

**Comment by aogriffiths** [comment]
Feel free to ignore this PR for now, while I resolve the comments on my first.
Adam

**Comment by aogriffiths** [comment]
This PR should be good to for you to review now. It can come after https://github.com/home-assistant/operating-system/pull/768 or replace it.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,52 @@
...
+# Home Assistant Operating System
 
-Home Assistant Operating System (HassOS) is based on [buildroot](https://buildroot.org/). It's a hypervisor for Docker and supports various kind of hardware. It is also available as virtual appliance for different virtualization solutions. The whole system is optimized for hosting [Home Assistant](https://www.home-assistant.io) and its features (to be precise, the [Add-ons](https://www.home-assistant.io/addons/)). You can update the system by using OTA updates or offline updates.
+Home Assistant Operating System (HassOS) is an operating system optimized for hosting [Home Assistant](https://www.home-assistant.io) and its [Add-ons](https://www.home-assistant.io/addons/).
 
-This is an embedded Linux which works different than a normal Linux distribution. The system is designed to run with minimal I/O and is optimized for its tasks.
+HassOS is uses Linux as a hypervisor for Docker, allowing each component of Home Assistant including add-ons to run in separate docker containers. It uses embedded Linux based on [buildroot](https://buildroot.org/), which is different to a "normal" Linux, and it is optimized for running Home Assistant on IoT devices.
```
Comment (agners) [inline README.md:5]: Also here, it should be Home Assistant Operating System (or OS) (see https://community.home-assistant.io/t/changing-the-home-assistant-brand/167500/16). I would drop the term "hypervisor", as this is more commonly used in the context of virtual machine. This is about containers.

Maybe:

```suggestion
Home Assistant Operating System uses Docker as Container engine. It by default deploys the Home Assistant Supervisor as a container. Home Assistant Supervisor in turn uses the Docker container engine to control Home Assistant Core and Add-Ons in separate containers. Home Assistant Operating System is **not** based on a regular Linux distribution like Ubuntu. It is built using [buildroot](https://buildroot.org/), a tool to build Linux based Operating Systems from source for embedded/IoT devices.
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,52 @@
...
+- Hardkernel ODROID
+- Intel NUC
+- Asus Tinker Board
+- Virtual appliances
+
+See full list and specific models [here]
+
+## Getting Started
+
+If you just want to use Home Assistant the official [getting started guide](https://www.home-assistant.io/getting-started/) and [installation instructions](https://www.home-assistant.io/hassio/installation/) take you through how to download HassOS and get it running on your machine.
```
Comment (agners) [inline README.md:27]: Replace HassOS with Home Assistant Operating System here, as well as in the sentence and title below.

File: Documentation/boards/hardkernel/odroid-xu4.md [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
-* It will be booting to uBoot (but no further). 
-    * If you have the serial adapter, you should be able to enter `distro_bootcmd` at the uboot prompt to continue booting.
-    * If not, flash the HassOS image to an SD card and boot off that temporarily (while the eMMC is also plugged in).
-* Once booted, login at the prompts and then enter `dd if=/dev/mmcblk0 of=/dev/mmcblk0boot0 bs=512 skip=63 seek=62 count=1440` at the linux prompt.
-* Reboot with eMMC (don't forget to flip the boot switch to eMMC)
-2) Clean/wiped/corruped boot sector:
-* You'll need to follow [Hardkernel's instructions](https://forum.odroid.com/viewtopic.php?f=53&t=6173) to get a working boot sector. Then flash HassOS and follow instructions above.
-* Alternatively, you can try flash HassOS to both an SD and eMMC, then boot off the SD with the eMMC also plugged in, then run `dd if=/dev/mmcblk1 of=/dev/mmcblk0boot0 bs=512 skip=1 seek=0 count=16381` at the Linux prompt. Note that this is untested, but in theory should work..
+The ODROID XU4 has a hidden boot sector that is only visible on the ODROID itself (can't be written by a card reader). There are a couple of possibilities:
```
[the remark this answers is not in the document]
Comment (aogriffiths) [reply Documentation/boards/hardkernel/odroid-xu4.md:5]: Changed see commit e257ce39bbd8f858dee9feeb0248644c6cb23b32

File: Documentation/boards/hardkernel/odroid-xu4.md [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
+
+1. **Working** e.g. the eMMC already had a working image before flashing HassOS:
+   - It will be booting to uBoot (but no further).
```
[the remark this answers is not in the document]
Comment (aogriffiths) [reply Documentation/boards/hardkernel/odroid-xu4.md:8]: Changed - see commit 64a418e4d5c1c8c73b6fadc0dad0299de5189f2c

File: Documentation/boards/hardkernel/odroid-xu4.md [hunk 5]
```diff
@@ -1,22 +1,24 @@
...
+     - If you have the serial adapter, you should be able to enter `distro_bootcmd` at the uboot prompt to continue booting.
+     - If not, flash the HassOS image to an SD card and boot off that temporarily (while the eMMC is also plugged in).
+   - Once booted, login at the prompts and then enter `dd if=/dev/mmcblk0 of=/dev/mmcblk0boot0 bs=512 skip=63 seek=62 count=1440` at the linux prompt.
+   - Reboot with eMMC (don't forget to flip the boot switch to eMMC)
+2. **Not Working** e.g. a clean/wiped/corruped boot sector:
```
[the remark this answers is not in the document]
Comment (aogriffiths) [reply Documentation/boards/hardkernel/odroid-xu4.md:13]: changed in commit 111d0dc3870b61e4d5c48bc087209b1fd21bbf57

File: Documentation/boards/intel/README.md [hunk 6]
```diff
@@ -1,9 +1,23 @@
 # Intel NUC
 
-This board configuration supports the Intel NUC mini PCs and compatibles. 
-Probably most recent computers will work.
+## Supported Hardware
+
+This board configuration supports the Intel NUC mini PCs and compatibles. Most recent MUC computers will probably work.
```
[the remark this answers is not in the document]
Comment (aogriffiths) [reply Documentation/boards/intel/README.md:5]: changed in commit 400348f658e11c626a93da05a4728c703411300a

File: README.md [hunk 7]
```diff
@@ -1,18 +1,52 @@
-# Home Assistant Operating system
+# Home Assistant Operating System
 
-Home Assistant Operating System (HassOS) is based on [buildroot](https://buildroot.org/). It's a hypervisor for Docker and supports various kind of hardware. It is also available as virtual appliance for different virtualization solutions. The whole system is optimized for hosting [Home Assistant](https://www.home-assistant.io) and its features (to be precise, the [Add-ons](https://www.home-assistant.io/addons/)). You can update the system by using OTA updates or offline updates.
+Home Assistant Operating System (HassOS) is an operating system optimized for hosting [Home Assistant](https://www.home-assistant.io) and its [Add-ons](https://www.home-assistant.io/addons/).
```
[the remark this answers is not in the document]
Comment (aogriffiths) [reply README.md]: see commit 61e7793

File: README.md (hunk 1 above)
Comment (aogriffiths) [reply README.md]: Merged these comments with previous comments on this paragraph. See commit 2190d41.

File: README.md (hunk 2 above)
Comment (aogriffiths) [reply README.md:27]: resolved in 57cbd3f

**Comment by aogriffiths** [comment]
hi @agners I think I've resolved all of your comments / suggestions.

File: Documentation/boards/raspberrypi/README.md [hunk 8]
```diff
@@ -1,15 +1,17 @@
...
+| Device              | Release Date  | Support         | Config             |
+|---------------------|---------------|-----------------|--------------------|
+| Raspberry Pi B/B+/A+|2012/2014/2014 | not recommended | [rpi](../../../buildroot-external/configs/rpi_defconfig)              |
+| Raspberry Pi 2 B    |2015           | not recommended | [rpi2](../../../buildroot-external/configs/rpi2_defconfig)             |
+| Raspberry Pi Zero   |2015           | not recommended | [rpi](../../../buildroot-external/configs/rpi_defconfig)              |
+| Raspberry Pi Zero W |2017           | not recommended | [rpi0_w](../../../buildroot-external/configs/rpi0_w_defconfig)           |
+| Raspberry Pi 3 B/B+ |2016/2018      | yes             | [rpi3](../../../buildroot-external/configs/rpi3_defconfig) / [rpi3_64](../../../buildroot-external/configs/rpi3_64_defconfig) |
+| Raspberry Pi 4 B    |2019           | yes*            | [rpi4](../../../buildroot-external/configs/rpi4_defconfig) / [rpi4_64](../../../buildroot-external/configs/rpi4_64_defconfig) |
+
+\*1,2 and 4 GiB versions of the Raspberry Pi 4 B are supported. The 8 GiB version is not yet supported
```
Comment (agners) [inline Documentation/boards/raspberrypi/README.md:14]: This changed very recently. I guess you need to rebase/merge the latest changes from the dev branch. But maybe its better if this gets merged first.

Comment (aogriffiths) [reply Documentation/boards/raspberrypi/README.md]: Made reference to this in #77ffa1f

**Pull request merged**